src/site/content/en/blog/is-it-modal/index.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Is it :modal?
+subhead: This handy CSS pseudo-selector gives you a way to select elements that are modal.
+authors:
+  - jheyy
+description: This handy CSS pseudo-selector gives you a way to select elements that are modal.
+date: 2022-09-01
+updated: 2022-09-01
+thumbnail: image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/xyyPgtSNjfzhcOGPn5lG.jpg
+alt: The unveiling of the new Torah display at Trinity International University, photo by Taylor Wilcox.
```

Comment:
where did the image come from? Rather than put credit in the alt, which isn't for credit it's for an image description. Add a line at the bottom with a link if there is one. Like we have on this post. https://web.dev/custom-properties-web-components/#private-and-public-custom-properties